app/src/viewmodels/home/MaximizeScreenViewModel.tsx

```diff
@@ -127,13 +128,15 @@ export function useMaximizeScreenViewModel(
         post_id: challenge.challenge_id ?? "",
         uid: currentUserId,
       };
+      setIsLoading(true);
       await appendComment(commentData);
       setCommentList([...commentList, commentData]);
       setCommentText("");
+      setIsLoading(false);
     }
   };
 
-  const postDate: any = challenge.date ? challenge.date : new Date();
+  const postDate: Date = challenge.date ? challenge.date : new Date();
```

Comment:
would this make the final date a new date for each user that maxes screen?

Reply:
I just forced the type to be correct, it shouldn't change anything functionally speaking.